Unix/disp/agentmgr.c

```diff
@@ -853,7 +853,7 @@ static pid_t _SpawnAgentProcess(
 
     trace_AgentLaunch_Failed(scs(agentProgram), errno);
     _exit(1);
-    return -1;  /* never get here */
+    // return -1;  /* never get here */
```

Comment:
if we successfully pbuild without this line I would suggest just deleting it instead of commenting it out or #ifdef-ing